pyproject.toml

```diff
@@ -44,25 +44,32 @@ hayhooks = "hayhooks.cli:hayhooks"
 source = "vcs"
 
 [tool.hatch.envs.default]
-dependencies = [
-  "coverage[toml]>=6.5",
-  "pytest",
-  "pytest-mock",
-]
-[tool.hatch.envs.default.scripts]
-test = "pytest {args:tests}"
-test-cov = "coverage run -m pytest {args:tests}"
+installer = "uv"
+
+[tool.hatch.envs.test.scripts]
+all = "pytest -vv {args:tests}"
+all-cov = "coverage run -m pytest {args:tests}"
 cov-report = [
   "- coverage combine",
   "coverage report",
 ]
 cov = [
-  "test-cov",
+  "all-cov",
   "cov-report",
 ]
 
+[tool.hatch.envs.test]
+installer = "uv"
```

Comment:
```suggestion
```

I think this line is not needed, since all environments inherit from `default`, unless they are self-referential or detached.

https://hatch.pypa.io/1.13/config/environment/overview/#inheritance